Python/bltinmodule.c

```diff
@@ -1976,9 +1976,15 @@ builtin_input_impl(PyObject *module, PyObject *prompt)
             promptstr = PyBytes_AS_STRING(po);
         }
         else {
-            po = NULL;
+            po = PyBytes_FromString("");
```

Comment:
actually I don't need this anymore for the change as mentioned the comment below.